code/__HELPERS/announcements.dm

```diff
@@ -16,20 +16,20 @@
  * * play_sound - if TRUE, play a sound with the announcement (based on player option)
  * * sound_override - optional, override the default announcement sound
  * * sender_override - optional, modifies the sender of the announcement
- * * encode_title - if TRUE, the title will be HTML encoded
- * * encode_text - if TRUE, the text will be HTML encoded
+ * * encode_title - if TRUE, the title will be HTML encoded (escaped)
+ * * encode_text - if TRUE, the text will be HTML encoded (escaped)
  * * color_override - optional, set a color for the announcement box
  */
 
-/proc/send_formatted_announcement(
+/proc/send_ooc_announcement(
 	text,
 	title = "",
 	players,
 	play_sound = TRUE,
-	sound_override = 'sound/ai/default/attention.ogg',
+	sound_override = 'sound/misc/bloop.ogg',
 	sender_override = "Server Admin Announcement",
 	encode_title = TRUE,
-	encode_text = TRUE,
+	encode_text = FALSE,
```

Comment:
Look how they massacred my boy

Reply:
this change allows HTML in announcements again. the encode_text arg strips HTML in the announcement